src/main/webapp/calendar-script.js

```diff
@@ -43,8 +66,95 @@ function getEvents(calendar) {
         title: event.name,
         start: event.strStartTime,
         end: event.strEndTime,
-        allDay: false
+        allDay: false,
+        extendedProps: {
+          stringDate: event.strStartTime,
+          address: event.address,
+          placeId: event.placeId, 
+          openTime: '9AM',
+          closeTime: '5PM',
+        }
       });
     });
   });
 }
+
+function createMap(modalBody, eventObj) {
+  const infoDisplay = document.createElement('div');
+  let address = document.createElement('p');
+  var urlFormatAddress = eventObj.extendedProps.address.replace(/\s/g, '+')
+  var url = 'https://www.google.com/maps/search/?api=1&query=' + urlFormatAddress + '&query_place_id=' + eventObj.extendedProps.placeId;
+  infoDisplay.innerHTML = '<b>Address: </b><a href=' + url + '>' + eventObj.extendedProps.address + '</a>' + '<br>';
```

Comment:
nit / personal preference: Add `target="_blank"` as an attribute to the `a` tag so that it opens in a new tab (https://www.w3schools.com/tags/att_a_target.asp). This way you still have the website open.